automatic/pencil/pencil.nuspec

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>pencil</id>
+    <version>3.0.4</version>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/pencil</packageSourceUrl>
+    <owners>chocolatey,Andrei Bejenaru</owners>
+    <title>Pencil</title>
+    <authors>Evolus</authors>
+    <projectUrl>http://pencil.evolus.vn</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/abejenaru/chocolatey-packages/16e4ddf69bd7dfc07597b7b3aaa7d8372eeec7fb/icons/pencil.png</iconUrl>
+    <copyright>(c) 2008-2017 Evolus</copyright>
```

Comment:
Is this the copyright showing in the software itself?

On the webpage I see `Copyright © 2008-2012 Evolus. All rights reserved.`